### ISSUE TITLE
Paper wallets should be discouraged

### ISSUE THREAD

**Issue by donnlee** [bitcoinbook/bitcoinbook#812]
```
I personally keep the vast majority of my bitcoin (99% or more)
stored on paper wallets, encrypted with BIP-38, with multiple
copies locked in safes. [Chapter 11]
```

I remember Andreas recommending AGAINST using paper wallets. This excerpt comes across as a testimonial for using paper wallets; any newb would likely seek/follow after reading this paragraph.

I can understand that paper wallets once served a purpose, especially when the book was written.  However, I was surprised to see this text not yet edited in the current branch.

**Comment by aantonop** [comment]
Yes, this should be removed and replaced with a warning about paper wallet
risks, now that we have much better methods of cold storage.

Thanks for noticing and bringing to my attention. Would you like to have a
go at writing up a pull request to fix this? At the very least let's remove
this in a standalone pull request. An additional pull request that provides
a warning and states explicitly that paper wallets are obsolete and risky
would be even better.

**Comment by donnlee** [comment]
@aantonop , thank you for the reply. I'd love to take a swing at editing this in a pull request. I'll work on it this week.

**Comment by Headbucket** [comment]
@aantonop I think it's a pity that the section 'paper wallets' in ch 1 has already been removed by the commit a3229bbbc0c929dc53ec11365051a6782695cb52.
I think a note about the dangers of paper wallets makes perfect sense. But in my opinion it is still a valid option for a small group of people: qualified software developers.

In my humble opinion, there are also certain risks with hardware wallets that could be pointed out (vulnerabilities, espionage, availability, ...)

Side note: Why did you remove a whole section in a documentation/refactoring commit like "bitcoin/Bitcoin capitalization changed everywhere"? Was it by accident?
